### ISSUE TITLE
Consistent naming for instantiated PenaltyModels

### ISSUE THREAD

**Issue by arcondello** [dwavesystems/penaltymodel#31]
should all be `pmodel` or `p_model` or similar. Not `widget`.

**Comment by arcondello** [comment]
```
bqm = dimod.BinaryQuadraticModel(__)
model = pm.PenaltyModel(__)
spec = pm.Specification(--)
```

**Comment by arcondello** [comment]
This needs to be updated in all of the documentation and code

**Comment by arcondello** [comment]
Fixed as part of version [1.0.0](https://github.com/dwavesystems/penaltymodel/releases/tag/1.0.0).
